sklbench/datasets/loaders.py

```diff
@@ -768,6 +768,33 @@ def load_road_network(
     return {"x": x, "y": y}, data_desc
 
 
+def load_from_datasetsroot(
+    data_name: str, dataset_params: Dict
+) -> Tuple[Dict, Dict]:
+    datasets_path = os.environ.get("DATASETSROOT")
+    if datasets_path and os.path.exists(datasets_path + "/workloads/" + data_name + "/dataset/" + dataset_params.get("train_file")):
+        train_df = pd.read_csv(datasets_path + "/workloads/" + data_name + "/dataset/" + dataset_params.get("train_file"))
+        x_train = train_df.iloc[:, :-1].to_numpy(dtype=np.float32)
+        y_train = train_df.iloc[:, -1].to_numpy(dtype=np.float32).reshape((-1, 1))
+        
+        if dataset_params.get("test_file"):
+            test_df = pd.read_csv(datasets_path + "/workloads/" + data_name + "/dataset/" + dataset_params.get("test_file"))

```

Comment:
You can probable use a variable for `datasets_path + "/workloads/" + data_name + "/dataset/"`